CleverPush/CleverPush/Source/CPAppBannerController.m

```diff
@@ -263,12 +263,12 @@ - (UIView*)composeButtonBlock:(CPAppBannerButtonBlock*)block {
     [button setContentCompressionResistancePriority:UILayoutPriorityRequired forAxis:UILayoutConstraintAxisVertical];
     [button handleControlEvent:UIControlEventTouchUpInside withBlock:^{
         self.actionCallback(block.action);
-        if (block.action.openInWebview) {
+        if (block.action.dismiss && block.action.openInWebview) {
```

Comment:
I don't 100% get this condition.

`block.action.dismiss` can work idependently of `block.action.openInWebview`

Why do we need to give the `dismiss` value to `openSafari` ?